central/metrics/info/gauge.go

```diff
@@ -44,7 +45,7 @@ func newGaugeVec() *prometheus.GaugeVec {
 			Name:      "info",
 			Help:      "A metric with a constant '1' value labeled by information identifying the Central installation",
 			ConstLabels: prometheus.Labels{
-				"central_id":      fetchInstallInfo().GetId(),
```

Comment:
Ideally, would be nice to return an error from the fetch install info function and make it clear that central_id will be empty on error. Also, what does that do to the data we're getting in telemetry

Reply:
If `central_id` is empty, then the corresponding labels in `rox_central_info` / `rox_sensor_info` would also be empty, and Prometheus would drop them. On the Telemeter backend the metric series would still be distinguished by the cluster ID of the OpenShift cluster.